docs/AboutUs.md

```diff
@@ -50,12 +50,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Tiong Meng Sheng
 
 <img src="images/johndoe.png" width="200px">
```

Comment:
now just add the image then youre set 👍

Reply:
agreed! please use a heart-stopping, blood-pumping, rizz-inducing picture of yourself!